.github/workflows/image.yml

```diff
@@ -56,15 +113,14 @@ jobs:
 
           # We push 3 tags to Dockerhub:
           # first, the full sha of the commit
-          docker tag "$IMAGE_URL" getsentry/taskbroker:${GITHUB_SHA}
-          docker push getsentry/taskbroker:${GITHUB_SHA}
+          docker buildx imagetools create --tag getsentry/taskbroker:${GITHUB_SHA} ghcr.io/getsentry/taskbroker:${{ github.sha }}
```

Comment:
If we've fetched the image from ghcr, why do we need to use `imagetools create` on it?

Reply:
Ah right. The docker pull should be removed first. See https://github.com/getsentry/symbolicator/pull/1716#discussion_r2143076552